packages/plugin-core/src/test/suite-integ/CompletionProvider.test.ts

```diff
@@ -0,0 +1,264 @@
+import { ENGINE_HOOKS } from "@dendronhq/engine-test-utils";
+import { describe } from "mocha";
+import { runLegacyMultiWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
+import _ from "lodash";
+import {
+  provideBlockCompletionItems,
+  provideCompletionItems,
+} from "../../features/completionProvider";
+import { VSCodeUtils } from "../../utils";
+import { CompletionItem, Position } from "vscode";
+import { NoteUtils } from "@dendronhq/common-all";
+import { expect } from "../testUtilsv2";
+import { NoteTestUtilsV4 } from "@dendronhq/common-test-utils";
+
+suite("completionProvider", function () {
+  let ctx = setupBeforeAfter(this, {
+    beforeHook: () => {},
```

Comment:
if you don't run anything before, you don't actually need the `beforeHook` method